pkg/bugzilla/client.go

```diff
@@ -0,0 +1,84 @@
+package bugzilla
+
+import (
+	"bytes"
+	"encoding/json"
+	"io/ioutil"
+	"log"
+	"net/http"
+)
+
+type Client interface {
+	ExecuteQuery(query string) (Bugs, error)
+}
+
+// httpBugzillaClient is a client object for the bugzilla API
```

Comment:
in general, no need to say `object` here.  We can already tell that it's an object by looking at the type declaration.  Just say `httpBugzillaClient is a client for the bugzilla API that connects via JSONRPC over HTTP` or somesuch.